src/components/inputs/password-input/password-input.visualroute.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
 import { PasswordInput } from 'ui-kit';
 import { Suite, Spec } from '../../../../test/percy';
 
 const value = 'hello world how are you?';
 
 export const routePath = '/password-input';
 
+const darkTheme = {
```

Comment:
Instead of repeating this in ever VRT test, we should define this once and we can define the theme provider at the app level directly.

I can spin up a separate PR to make it more clear

Reply:
https://github.com/commercetools/ui-kit/pull/636

Reply:
Yeah good idea 🎉